migrations/dlu/12_modular_build_ugc.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS ugc_modular_build (
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	ugc_id BIGINT NOT NULL,
```

Comment:
Why is Id and ugc_id needed? Ugc_id should be unique and can serve as a primary key